### ISSUE TITLE
Cant get lvl 70 boost to work

### ISSUE THREAD

**Issue by SgtHolst** [dalaranwow/dalaran-wow#5663]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. i have logged out and closed the game.
2. i dont have any character changes available and i have bought 22 donation point.
3. but it still wont let me buy a lvl 70 boost. my account name is SgtHolst.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by lordmatthias** [comment]
I do am having the same issue, when I select the character select free instant 70 and click next nothing happens. Screenshot attached.
![issue](https://user-images.githubusercontent.com/46628000/51074862-f2984580-1649-11e9-87d6-9cfbff2ee4bb.PNG)

**Comment by Golganeth** [comment]
Select the "Free Instant 70" option. If it becomes blue it will work, otherwise it will just remain white/grey and unselected.
Closing

**Comment by lordmatthias** [comment]
Hi Golganeth,

I tried selecting the field, is turns blue, I select next step, and nothing happens (no loading page or any sign that anything actually happened).

**Comment by Golganeth** [comment]
What Internet Browser are you trying it with? Tried it with Chrome and Firefox and no issues at all, in any case make sure your browser is up to date, a pop-up should happen once you select "next step", letting you choose the gear and all that.

If the issue persists and you are trying a different browser, please try with the above browsers and delete your Cache. Try also logging out your account and logging back in. Let me know if it helped

**Comment by lordmatthias** [comment]
At the time I tried using Chrome, Edge, and Firefox. Your suggestion about popup blocking did trigger a thought, it looked it appears that google is flagging the site as insecure and was disabling a lot of content. I just checked to allow the content to load and now the page loads normally and the boost appears to work. 

So my issue is resolved (thank you), but I have attached the warning message I get, it appears your cert is valid until later this year so maybe, by chance are you leveraging TLS 1.0 or an older SSL version? I suspect a large number of users are probably running into this issue.
